frontend/src/metabase/admin/performance/utils.tsx

```diff
@@ -315,3 +315,13 @@ export const translateConfigFromAPI = (config: CacheConfig): CacheConfig =>
 /** Translate a config from the frontend's format into the API's preferred format */
 export const translateConfigToAPI = (config: CacheConfig): CacheConfig =>
   translateConfig(config, "toAPI");
+
+export const getDefaultValueForField = (
+  strategyType: CacheStrategyType,
+  fieldName?: string,
+) => {
+  const schema = getStrategyValidationSchema(
+    PLUGIN_CACHING.strategies[strategyType],
+  );
+  return fieldName ? schema.cast({})[fieldName] : "";
+};
```

Comment:
Moved this function here from `frontend/src/metabase/admin/performance/components/StrategyForm.tsx`